PeakLocalisations/Localisation/Blocks/Resolutions.xcstrings

```diff
@@ -101,6 +131,12 @@
             "state" : "translated",
             "value" : "You're in the home stretch! Just one more goal to go"
           }
+        },
+        "es" : {
+          "stringUnit" : {
+            "state" : "translated",
+            "value" : "¡Está en la recta final! Solo falta un gol más"
```

Comment:
i'd use meta here too like on the rest of sentences. my first thought reading gol was a literal soccer goal!

Reply:
+1